check_stability.py

```diff
@@ -726,6 +787,8 @@ def main():
     global wpt_root
     global wptrunner_root
     global logger
+    global product_args
```

Comment:
Instead of the globals here, let's pass the args in as kwargs to the `Browser` class i.e. `browser_cls(vars(args))` below. Then the base implementation can ignore these args, and the sauce browser can pick out the ones it needs in its `__init__` method.